.github/workflows/awscleanup.yaml

```
@@ -0,0 +1,81 @@
+name: Daily AWS Cleanup Bot
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+    branches:
+      - test-awsresourcecleanup
+  push:
+    branches:
+      - test-awsresourcecleanup
+
+jobs:
+  cleanup:
+    runs-on: linux-amd64-cpu4
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up AWS CLI
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-west-1
+
+      - name: Identify resources for deletion
+        id: identify-resources
+        run: |
+          # Find vpcs with names ci*
+          vpcs=$(aws ec2 describe-vpcs \
+            --filters "Name=tag:Name,Values=ci*" \
+            --query "Vpcs[].VpcId" \
+            --output text | tr -d '\r' | tr '\n' ' ')
+          echo "Found VPCs: $vpcs"
+          echo "vpcs=$vpcs" >> $GITHUB_ENV
```

env vars should be in uppercase
```suggestion
          echo "vpcs=$vpcs" >> $GITHUB_ENV
```

---

the env var can be named `AWS_VPC_IDS`